config/routes.rb

```diff
@@ -493,15 +493,23 @@
   end
 
   unless Rails.env.production?
-    namespace :lti_deployment do
-      get 'get_canvas_config'
-      post 'launch'
-      get 'public_jwk'
-      post 'redirect_login'
-      get 'choose_course'
-      post 'choose_course'
-      post 'create_course'
-      post 'create_lti_grades'
+    resources :lti_deployment, only: [] do
+      collection do
+        get 'public_jwk'
+        get 'choose_course'
```

Comment:
I'm wondering why `choose_course` is a collection route instead of a member route. The session carries the `lti_deployment_id` at this point; is there a reason that's better than keeping the `lti_deployment_id` in the URL?

Reply:
You're right - at the point we get to `choose_course` there's no reason not to put the `lti_deployment_id` in the url